observability/cluster_observability_operator/ui_plugins/dashboard-ui-plugin.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="dashboard-ui-plugin"]
+= Dashboard UI plugin
+include::_attributes/common-attributes.adoc[]
+:context: dashboard-ui-plugin
+
+toc::[]
+
+:FeatureName: The Cluster Observability Operator
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The dashboard UI plugin supports enhanced dashboards in the OpenShift web console at *Observe* -> *Dashboards* . You can add other Prometheus datasources from the cluster to the default dashboards, in addition to the in-cluster datasource. This results in a unified observability experience across different data sources.
+
+The plugin will search for datasources as `ConfigMaps` in the `openshift-config-managed` namespace, that have the label `console.openshift.io/dashboard-datasource: 'true'`. 
```

Comment:
Not sure if the following is better 
The plugin will search for datasources from `ConfigMaps`

Reply:
Agreed and done.